python/triton/tools/aot.py

```diff
@@ -7,7 +7,7 @@
 if __name__ == '__main__':
 
     # valid source and target formats
-    VALID_FORMATS = ['triton-ir', 'triton-gpu-ir', 'llvm-ir', 'ptx', 'amdgcn']
+    VALID_FORMATS = ['triton-ir', 'triton-ptx-gpu-ir', 'llvm-ptx-ir', 'ptx', 'amdgcn']
```

Comment:
I propose to separate ptx and gcn sets of targets, because there is no such thing as "compute capability" for AMD GPU, as far as I know. So `sm` option make sense only for ptx.